schema/message-groups.schema.json

```diff
@@ -0,0 +1,73 @@
+{
+  "title": "MessageGroup",
+  "description": "Configuration object for groups of Messaging System messages",
+  "type": "object",
+  "version": "1.0.0",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "A unique identifier for the message that should not conflict with any other previous message."
+    },
+    "enabled": {
+      "type": "boolean",
+      "description": "Enables or disables all messages associated with this group."
+    },
+    "userPreferences": {
```

Comment:
Out of curiosity, how would `enable` and `userPreferences` work together in practice?

Reply:
* `enable` allows us to turn off all messages that define `groups: ['messaging-experiments']`
* `userPreferences` (array of preferences) turns `enabled` to false when any prefs are false.